Bottom: 7ff312579e0d5e58293bfa4bf19b2a1d6b6e0c86
Top:    8a37bb5b752570cd64b1bdb2562cf2182fb6c357
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 22:27:26 +0000

Refresh of more-changes

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 3e7181d..ef11dd8 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1063,10 +1063,7 @@ vertex_in_seg_rounded (const BoxType * b, void *cl)
    * already been intersected this pass, skip it until the next pass.
    */
   if (s->intersected)
-    {
-//      g_warning ("Should not reach this 1");
-      return 0;
-    }
+    return 0;
 
   if (!line_point_inters (EDGE_BACKWARD_VERTEX (s->v)->point, EDGE_FORWARD_VERTEX (s->v)->point, i->v->point))
     return 0;
